docs/manual_install.rst

```diff
@@ -40,26 +40,19 @@ Bot Settings:
 Database
 --------
 
-Create a new mysql database and user for your bot.
+Create a new mysql database and user for your bot, granting full priviledges for the db. For example:
 
-Command to access the terminal as root user: ``mysql -u root -p``
+.. code-block:: shell
+    mysql -u root -p
+    MariaDB [(none)]> CREATE USER 'username'@'localhost' IDENTIFIED BY 'PASSWORD';
+    MariaDB [(none)]> CREATE DATABASE databasename;
```

Comment:
`MariaDB [(none)]>` do we want these here?

Reply:
ACK

Reply:
Good point, simplified this quite a bit.